kubernetes-client/src/main/java/io/fabric8/kubernetes/client/utils/Utils.java

```diff
@@ -44,6 +44,8 @@
 
 import io.fabric8.kubernetes.client.KubernetesClientException;
 
+import javax.swing.text.html.Option;
```

Comment:
Umm, what's this used for?

Reply:
:man_facepalming: nothing, IDE autoimport and me not checking  :man_facepalming: :man_facepalming: :man_facepalming: